### DISCUSSION TITLE
Nested Layout

### DISCUSSION THREAD

**Discussion by ghost** [refinedev/refine#5586] [Q&A] [answered]
In Nextjs, it is supported to use nested layouts. In refine, I've found that inside the `<Refine>` tag, everything is rendered. Also a main basic theme. Is it possible to use multiple or nested layouts in Refine?

Thanks in advance.

**Comment by BatuhanW** [comment, marked as answer]
Hey @aislam1234, `<Refine />` component doesn't render anything, but provides context for other refine components. You can use nested layouts normally as you would do it with NextJS projects and with your UI library.

**Reply by ghost** [reply]
@BatuhanW Okay, thanks.
